android-components/components/browser/state/src/test/java/mozilla/components/browser/state/engine/middleware/TranslationsMiddlewareTest.kt

```diff
@@ -386,6 +387,76 @@ class TranslationsMiddlewareTest {
         waitForIdle()
     }
 
+    @Test
+    fun `WHEN FetchTranslationDownloadSize is requested AND succeeds THEN SetTranslationDownloadSize is dispatched`() = runTest {
+        val translationSize = TranslationDownloadSize(
+            fromLanguage = Language("en", "English"),
+            toLanguage = Language("fr", "French"),
+            size = 10000L,
+            error = null,
+        )
+
+        val action =
+            TranslationsAction.FetchTranslationDownloadSizeAction(
+                tabId = tab.id,
+                fromLanguage = translationSize.fromLanguage,
+                toLanguage = translationSize.toLanguage,
+            )
+        translationsMiddleware.invoke(context, {}, action)
```

Comment:
I would suggest adding the named parameters here and in general. It will make refactoring easier in the future when it's not obvious what a long list of parameters map to.

Reply:
Updated, agreed!